test/files.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+
+module.exports = (common) => {
+  describe('.file', () => {
+    let ipfs
+
+    before((done) => {
+      common.setup((err, _ipfs) => {
+        expect(err).to.not.exist
+        ipfs = _ipfs
+        done()
+      })
+    })
+
+    after((done) => {
+      common.teardown(done)
+    })
+
+    describe('.cat', () => {
+      it('returns file stream', (done) => {
+        const hash = 'QmT78zSuBmuS4z925WZfrqQ1qHaJ56DQaTfyMUF7F8ff5o'
+        ipfs.files.cat(hash, (err, res) => {
+          expect(err).to.not.exist
+          res.on('data', (data) => {
```

Comment:
I'm confused here, ```stream.on('data', (file) => {``` 
should be labelled  ```stream.on('data', (stream) => {```?

But that is already labelled for the object stream that the core returns. I think you may be wanting the core to be changed like mention <a href="https://github.com/ipfs/interface-ipfs-core/pull/21/files#r64141277">here</a> ?